src/main/java/frc/robot/commands/pathplanning/ComboShot.java

```diff
@@ -0,0 +1,18 @@
+package frc.robot.commands.pathplanning;
+
+import edu.wpi.first.wpilibj2.command.ParallelCommandGroup;
+import frc.robot.commands.feeder.FeederGamepieceUntilLeave;
+import frc.robot.subsystems.Feeder;
+import frc.robot.subsystems.Shooter;
+
+public class ComboShot extends ParallelCommandGroup {
+    public ComboShot(Shooter shooter, Feeder feeder) {
+        addCommands(
+                new BasicShoot(shooter,1),
+                new FeederGamepieceUntilLeave(feeder)
+        );
+        addRequirements(shooter,feeder);
```

Comment:
no need to add requirements for sequences, it is implied.
Also - once everything is working, we want to add logging.

Reply:
Should I add logging when we instantiate the object in robot container or in the sequence itself?